src/mixins/Projector.ts

```diff
@@ -286,6 +288,24 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(Base: T)
 			return this._projection.domNode.outerHTML;
 		}
 
+		_afterRootCreateCallback(element: Element,
```

Comment:
formatting a bit weird here (and the other functions)

Reply:
done